app/models/vote.rb

```diff
@@ -1,21 +1,9 @@
 class Vote < ActiveRecord::Base
-  VALID_VALUES = [-2, -1, 1, 2]
-  VOTE_TYPES = [
-    ["I hate it", -2].freeze,
-    ["I'm against it", -1].freeze,
-    ["I like it", 1].freeze,
-    ["I love it", 2].freeze
-  ].freeze
+  enum value: { hate_it: -2, dislike_it: -1, like_it: 1, love_it: 2 }
 
-  validates :value, presence: true
+  validates :value, presence: true, inclusion: { in: Vote.values.keys }
```

Comment:
Is this inclusion validation required? I would assume that `enum` does it out of the box, but I'm just making assumptions.